README.md

```diff
@@ -89,39 +89,69 @@ In our hand-eye calibration we use the following frames:
 
 All our tools can either be run via ROS, using
 
-```
+```bash
 rosrun hand_eye_calibration <tool>.py [arguments]
 ```
 
 or directly by changing into this directory (e.g. `~/catkin_ws/src/hand_eye_calibration/hand_eye_calibration`) and executing:
 
-```
+```bash
 ./bin/<tool>.py [arguments]
 ```
 
+### Step-by-Step Calibration
+
 A typical use case consists of the following steps (here using ROS):
 
 - Extract poses from tf (ROS transformation type) messages (with time stamps):
-  ```
+  ```bash
   rosrun hand_eye_calibration tf_to_csv.py --bag calibration.bag --tf_source_frame end_effector --tf_target_frame base_link --csv_output_file tf_poses_timestamped.csv
   ```
 - Extract poses from images (with time stamps):
-  ```
-  rosrun hand_eye_calibration target_extractor_interface.py --bag calibration.bag --calib_file_camera calib/camera_intrinsics.yaml --calib_file_target calib/target.yaml --image_topic /camera/rgb/image_raw --output_file camera_poses_timestamped.csv
+  ```bash
+  rosrun hand_eye_calibration target_extractor_interface.py \
+    --bag calibration.bag \
+    --calib_file_camera calib/camera_intrinsics.yaml \
+    --calib_file_target calib/target.yaml \
+    --image_topic /camera/rgb/image_raw \
+    --output_file camera_poses_timestamped.csv
   ```
 - Time alignment of the poses and interpolate the two sets at given time stamps:
-  ```
-  rosrun hand_eye_calibration compute_aligned_poses.py --poses_B_H_csv_file tf_poses_timestamped.csv --poses_W_E_csv_file camera_poses_timestamped.csv --aligned_poses_B_H_csv_file tf_aligned.csv --aligned_poses_W_E_csv_file camera_aligned.csv
+  ```bash
+  rosrun hand_eye_calibration compute_aligned_poses.py \
+    --poses_B_H_csv_file tf_poses_timestamped.csv \
+    --poses_W_E_csv_file camera_poses_timestamped.csv \ --aligned_poses_B_H_csv_file tf_aligned.csv \
+    --aligned_poses_W_E_csv_file camera_aligned.csv \
+    --time_offset_output_csv_file time_offset.csv
   ```
 - Perform the dual-quaternion-based hand-eye calibration:
-  ```
-  rosrun hand_eye_calibration compute_hand_eye_calibration.py --aligned_poses_B_H_csv_file tf_aligned.csv --aligned_poses_W_E_csv_file camera_aligned.csv --visualize True
+  ```bash
+  rosrun hand_eye_calibration compute_hand_eye_calibration.py \
+    --aligned_poses_B_H_csv_file tf_aligned.csv  \
+    --aligned_poses_W_E_csv_file camera_aligned.csv \
+    --time_offset_output_csv_file time_offset.csv \
```

Comment:
Isn't this an input parameter here? Maybe call it input or just `time_offset_csv_file`

Reply:
agreed, this is wrong, will fix it